packages/leaders-program/src/components/card/blocks/company-details.vue

```diff
@@ -76,6 +81,9 @@ export default {
     emitProfileClick(sourceLabel, data, event) {
       this.$emit('profile-click', { sourceLabel, ...data }, event);
     },
+    translate(key) {
+      return i18n(this.lang, key);
```

Comment:
@Shinsina up for convo, but based on other do we also need to add the if land !== en check here?